### PR TITLE
Style Shared albums page 

### PR DESCRIPTION
**Pull request by SophieSjoberg**
### Description 
- Add carousel to display recently added photos on "Shared albums" page 
- Display all family albums
- Add styling of album titles
- Adds style to albums show page

### Screenshot
screen shot 2018-02-10 at 11 06 32

### CODE REVIEW

File: features/users_can_share_photos_with_family_on_family_album_page.feature [hunk 1]
```diff
@@ -13,14 +13,16 @@ Feature: User can share photos with family
       | lisa@gmail.com | heythere123 | heythere123           | The Simpsons |
 
     Given the following image exist in album
-      | title     | image_file_name | album_title       |  family     |
-      | Something | random.png      | Lisas renovation  | The Simpsons|
+      | title     | image_file_name | album_title       |  family       |
```
Comment (NegativeKarma) [inline features/users_can_share_photos_with_family_on_family_album_page.feature]: fix the indentation of family

File: app/controllers/albums_controller.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@ class AlbumsController < ApplicationController
 
   def index
     @albums = current_user.family.albums
+    @recent_photos = @albums.map(&:photos).flatten.sort_by(&:created_at).last(3)
```
Comment (NegativeKarma) [inline app/controllers/albums_controller.rb:7]: Did you create this one with Raoul?

Comment (SophieSjoberg) [reply app/controllers/albums_controller.rb:7]: yepp, we can go through it if you want

Comment (NegativeKarma) [reply app/controllers/albums_controller.rb:7]: No, it's ok

**Pull request merged**